.github/workflows/ci.yaml

```diff
@@ -25,4 +25,11 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - run: npm ci
     - run: npm run build --if-present
-    - run: npm test
\ No newline at end of file
+    - run: npm run test-coverage
+
+    - name: Publish Code Coverage Results
+      uses: coverallsapp/github-action@master
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do you need me to do anything with authorizing the token?

Reply:
This token is an automatic token that comes from GH Actions so no need to add/approve anything